label_sync/labels.yaml

```diff
@@ -744,7 +744,7 @@ repos:
           - name: area/release-infra
         target: both
         addedBy: label
-      - color: ededed
+      - color: b60205
```

Comment:
We don't really block merges on `needs-priority`, so the colour change isn't "technically" applicable here.

I see most folks ignore this label in k/k and for PRs where the priority label is indeed applicable (critical-urgent in case of an important bug), people add it anyway. Can we only automatically apply this label during code freeze (as before)?

We only "require" this label during code freeze: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md#tldr

Reply:
Can we limit merge blocking labels to red, and make others some other color, like orange? Not sure if needs-kind is blocking.

Reply:
>  Not sure if needs-kind is blocking.

`needs-kind` is blocking today. :)

Reply:
:wave: bump

Reply:
needs-kind and needs-sig are blocking, but only for kubernetes/kubernetes, I opted to pull back on changing them